pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -2334,6 +2355,101 @@ def _impedance_calcul(self):
+            `0` places the probe at the center of the patch.
+            `1` places the probe at the edge of the patch.
+            Default: 0
+
+        rel_y_offset : float, value between 0 and 1
+            `0` places the probe at the center of the patch.
+            `1` places the probe at the edge of the patch.
+            Default: 0
+
+        d : float, probe diameter
+            Default: 0.01
+
+        name : str, optional name of probe port.
+            Default value `"Probe"`
+
+        Returns
+        -------
+        bool
+
+        Examples
+        --------
+
+        >>> from pyaedt import Hfss
+        >>> from pyaedt.modeler.stackup_3d import Stackup3D
+        >>> hfss = Hfss()
+        >>> my_stackup = Stackup3D(hfss, 2.5e9)
+        >>> gnd = my_stackup.add_ground_layer("gnd")
+        >>> my_stackup.add_dielectric_layer("diel1", thickness=1.5, material="Duroid (tm)")
+        >>> top = my_stackup.add_signal_layer("top")
+        >>> my_patch = top.add_patch(frequency=None, patch_width=51, patch_name="MLPatch")
+        >>> my_stackup.resize_around_element(my_patch)
+        >>> my_patch.create_probe_port(gnd)
+        """
+        probe_or = 2.45 * r  # Probe outer radius (relative to inner radius).
+        feed_length = 6 * probe_or
+
+        # Add parameters for relative probe feed location.
+        self.application["probe_x_rel"] = str(rel_x_offset)
+        self.application["probe_y_rel"] = str(rel_y_offset)
+        x_probe = self.position_x.name + "+(1.0 + probe_x_rel)*" + self.length.name + " / 2"
+        y_probe = self.position_y.name + "+  probe_y_rel *" + self.width.name + " / 2"
+
+        probe_height = (
+            self._signal_layer.elevation.name
+            + " - "
+            + reference_layer.elevation.name
+            + " - "
+            + reference_layer.thickness.name
+        )
+        z_ref = reference_layer.elevation.name + " + " + reference_layer.thickness.name
+        probe_pos = [x_probe, y_probe, z_ref]  # Probe base position.
+        probe_wire = self.application.modeler.create_cylinder(
+            cs_axis="Z", position=probe_pos, radius=r, height=probe_height, name=name, matname="copper"
+        )
+        probe_feed_wire = self.application.modeler.create_cylinder(
+            cs_axis="Z", position=probe_pos, radius=r, height=-feed_length, name=name + "_feed_wire", matname="copper"
+        )
+        probe_feed_outer = self.application.modeler.create_cylinder(
+            cs_axis="Z",
+            position=probe_pos,
+            radius=probe_or,
+            height=-feed_length,
+            name=name + "_feed_outer",
+            matname="vacuum",
+        )
+
+        # Probe extends through the ground plane.
+        self.application.modeler.subtract(reference_layer.name, probe_feed_outer.name)
+
+        # Find face on probe with max area. This is the outer ground and will be assigned PEC.
```

Comment:
```suggestion
        # Find face on probe with max area. This is the outer ground and is assigned PEC.
```
I've seen earlier when it is pec (in lower case).  Should it be pec or PEC. We should standardize.